pkgs/astral-sh/ty/registry.yaml

```diff
@@ -22,3 +25,6 @@ packages:
         overrides:
           - goos: windows
             format: zip
+            files:
+              - name: ty.exe
+                src: ty.exe
```

Comment:
- `.exe` should not be appended to `.files[].name`
- By default aqua appeends `.exe` in Windows, so you don't need to append `.exe` to `.files[].src`
- If `src` is equivalent to `name`, please omit `src`

Reply:
That's good to know!

Thank you for the clarification, and fixing up my mistake 🙂